.murdock

```diff
@@ -5,6 +5,8 @@
 # and / or
 #export APPS="examples/hello-world tests/unittests"
 
+QUICKBUILD_BOARDS="nrf52840dk samr21-xpro esp32-wroom-32 native"
```

Comment:
Why not use the board tiers here? This would IMO be the cleaner approach, as we could have a bit more control on the coverage. E.g. board tier 0 would be pretty close to the 5 minutes, board tier 1 should be like 15 minutes build, etc.

Reply:
That just requires `FEATURES_REQUIRED=board_tier_0` before executing `make info-boards-supported`, right?

Reply:
exactly